### PR TITLE
feat(url): support url to redirect or call API in background

### PR DESCRIPTION
**Pull request by bjagg**
This requires NotificationsPortlet 4.6.0

### CODE REVIEW

File: @uportal/notification-modal/package.json [hunk 1]
```diff
@@ -24,13 +24,17 @@
         "@uportal/open-id-connect": "1.30.1",
         "axios": "0.19.0",
         "bootstrap-vue": "2.0.0-rc.21",
-        "vue": "2.6.10"
+        "eslint": "^6.1.0",
+        "eslint-plugin-prettier": "^3.1.0",
+        "eslint-plugin-vue": "^5.2.3",
```
Comment (ChristianMurphy) [inline @uportal/notification-modal/package.json]: Could `eslint-*` dependencies be moved to `devDependencies`?

File: @uportal/notification-modal/package.json [hunk 2]
```diff
@@ -24,13 +24,17 @@
...
+        "jquery": "^3.4.1",
```
Comment (ChristianMurphy) [inline @uportal/notification-modal/package.json]: Does this component depend on `jQuery`? I'm not spotting where it's used.

File: package.json [hunk 3]
```diff
@@ -139,5 +139,7 @@
         "node": "^10.0.0 || ^11.0.0",
         "npm": "^6.5.0"
     },
-    "dependencies": {}
+    "dependencies": {
+        "eslint-plugin-prettier": "^3.1.0"
```
Comment (ChristianMurphy) [inline package.json]: Could this be moved to `devDependencies`?

**Comment by bjagg** [comment]
Okay, cleaned up those package.json files!

**Pull request merged**